src/containerapp/azext_containerapp/_utils.py

```diff
@@ -1594,17 +1592,17 @@ def create_extension(cmd, extension_name='containerapps-ext', connected_cluster_
     cluster_name = parsed_extension.get("name")
 
     e = models.Extension()
+    e.identity = models.Identity(type="SystemAssigned")
```

Comment:
why do we need the identity ?

Reply:
Keep consistent with 'az k8s-extension create' command.